### DISCUSSION TITLE
Why is there a 4.4.8 version at majorgeeks?

### DISCUSSION THREAD

**Discussion by GunnzAkimbo1974** [Open-Shell/Open-Shell-Menu#1098] [General]
Doesn't seem legit.

**Comment by blackcrack** [comment]
it's a release from @Ibuprophen repository..
any one dummbatz have post the packet from Ibuprophen :\

**Comment by Ibuprophen** [comment]


**Reply by YesTrustMe** [reply]
Care to say what your best guesses are? Is OpenShell being abandoned or moving to a pay for use model? Surely you must have some idea why they suddenly excluded you without an explanation.

**Comment by ge0rdi** [comment]
Wrong release was made by mistake.

**Comment by blackcrack** [comment]
the original File is them of the original Repository
and this is at moment 4.4.173 !!
https://ci.appveyor.com/project/passionate-coder/open-shell-menu/build/artifacts
and not from an foreign Repository @Ibuprophen and also not from your Repository Ibu !!
best

**Comment by ge0rdi** [comment]
> I had uploaded the v4.4.200 to my File Host Account for others to test to see if the installation error is resolved (hopefully).

Official versions are available via GitHub releases page (or AppVeyor in case of nightly builds).
We do not recommend to download/install from unofficial sources.

**Reply by YesTrustMe** [reply]
> > I had uploaded the v4.4.200 to my File Host Account for others to test to see if the installation error is resolved (hopefully).
> 
> Official versions are available via GitHub releases page (or AppVeyor in case of nightly builds). We do not recommend to download/install from unofficial sources.

Majorgeeks is about as official as it gets for a third party site, though.

**Comment by XenHat** [comment]
As far as I'm aware, we haven't had an official release in a while. If someone submitted Open-Shell to MajorGeeks, I was not informed.

Please only download software from the authors' official release channel (Open-Shell being here, on https://github.com/Open-Shell/Open-Shell-Menu ).
